src/System.Security.Cryptography.Pkcs/src/Interop/Windows/Crypt32/Interop.CryptMsgType.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Text;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+using Microsoft.Win32.SafeHandles;
+
+internal static partial class Interop
+{
+    internal static partial class Crypt32
+    {
+        internal enum CryptMsgType : int
```

Comment:
Eh… looks like by the time I needed it the second time, I’d forgotten I’d created it the first time. Those can be unified.